.eslintrc.js

```diff
@@ -197,8 +197,8 @@ module.exports = {
 
         // Packages that have switched to ESM sources:
         'packages/@uppy/audio/src/**/*.js',
-        'packages/@uppy/aws-s3/src/**/*.js',
         'packages/@uppy/aws-s3-multipart/src/**/*.js',
+        'packages/@uppy/aws-s3/src/**/*.js',
```

Comment:
Not sure about this change, when I run `ls packages/@uppy`, `aws-s3` is listed before `aws-s3-multipart`.

Reply:
I executed the command `>sort lines` in VSCode.